jormungandr/src/intercom.rs

```diff
@@ -500,7 +502,12 @@ pub fn stream_request<T, R>(
 #[allow(clippy::large_enum_variant)]
 #[derive(Debug)]
 pub enum TransactionMsg {
-    SendTransaction(FragmentOrigin, Vec<Fragment>),
+    SendTransactions {
+        origin: FragmentOrigin,
+        fragments: Vec<Fragment>,
+        fail_fast: bool,
```

Comment:
This currently seems redundant with `FragmentOrigin`, but it makes the code clearer.